scala/compiler-jps/src/org/jetbrains/jps/incremental/scala/local/worksheet/ILoopWrapperFactoryHandler.scala

```diff
@@ -108,7 +108,7 @@ object ILoopWrapperFactoryHandler {
 
   private val JAVA_USER_CP_KEY = "java.class.path"
   private val STOP_WORDS = Set("scala-library.jar", "scala-nailgun-runner.jar", "nailgun.jar", "compiler-shared.jar",
-    "incremental-compiler.jar", "compiler-jps.jar")
+    "incremental-compiler.jar", "compiler-jps.jar", "hydra-compiler-jps.jar")
```

Comment:
In theory, it would be best if we use the vanilla Scala compiler in the worksheet (it's only one source file). However, I'm not sure that's actually possible, is compilation of the worksheet file delegate to Zinc (and hence it will use the set compiler-bridge) or do you handle compilation of the worksheet file outside of Zinc?

Reply:
I suggest to leave it as is right now. Worksheet uses zinc, but I don't think it will be a problem.

Reply:
Ok. But how should the `hydra-compiler-jps.jar` be added to it? Or are you suggesting we leave the code above as it is now?

Reply:
Yes, you may leave it here. I think this part should be refactored, but I'm not sure yet how exactly to do it.